css-four-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+body {
+  background-color: rgb(247, 248, 250);
+  width: 70vw;
+  min-width: 800px;
+  font-family: 'Barlow', sans-serif;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.row {
+  width: 85.7%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.column-fourth {
+  width: 23%;
+}
+
+.screen {
+  width: 100%;
+}
+
+.icon {
+  width: 23%;
+}
+
+h2 {
+  font-size: 1.28vw;
+  font-weight: 800;
+}
+
+h6 {
+  font-size: 0.75vw;
```

Comment:
you want to avoid using the responsive units (vw, vh, vmin, vmax) for font sizes. It is better to go with rem, em, or px

Reply:
Thanks for your review. Fixed accordingly.